.github/workflows/pipeline.yml

```
@@ -0,0 +1,96 @@
+
+      - name: Load SSH key and deploy to EC2
+        uses: webfactory/ssh-agent@v0.8.1
+        with:
+          ssh-private-key: ${{ secrets.EC2_SSH_KEY }}
+
+      - name: Deploy to EC2 (via ssh)
+        env:
+          EC2_HOST: ${{ secrets.EC2_HOST }}
+          EC2_USER: ${{ secrets.EC2_USER }}
+          S3_BUCKET: ${{ secrets.S3_BUCKET }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:33: Double quote to prevent globbing and word splitting

(shellcheck)